lib/netsuite/configuration.rb

```diff
@@ -15,6 +15,8 @@ def attributes
     end
 
     def connection(params={}, credentials={})
+      check_credentials(credentials)
```

Comment:
Why were these changes required in your case? This smells like something specific to your application and not something we'd want to include in the netsuite gem.